yaguzmang/ghg-emission-calculation-tool
# Initialise Strapi

Install Strapi.

## How to test:

```
cd backend
npm run develop
```

You should be directed to http://localhost:1337/admin to set up your admin user account. Create the admin user and you should see the Strapi dashboard.

<img width="1500" alt="Screenshot 2023-03-30 at 11 59 42" src="https://user-images.githubusercontent.com/18427577/228785144-b38ecb1d-8308-4da6-aea7-f9cc8ed2af53.png">
